### ISSUE TITLE
Found interface org.apache.poi.util.POILogger, but class was expected

### ISSUE THREAD

**Issue by n-krueger** [HL7/fhir-ig-publisher#464]
## Affected Versions
- 1.1.123
- 1.1.124

## Affected Platforms
- macOS 12.4, x86_64, OpenJDK 17.0.3
- Ubuntu 20.04, x86_64, OpenJDK 17.0.3

Windows was not tested.

## Expected Behavior
`java -jar $publisher -ig .` succeeds in FHIR Shorthand project.

## Actual Behaviour
`java -jar $publisher -ig .` fails in FHIR Shorthand project with Exception:

```
Exception in thread "main" java.lang.IncompatibleClassChangeError: Found interface org.apache.poi.util.POILogger, but class was expected
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:497)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1489)
	at org.apache.poi.ooxml.POIXMLDocument.write(POIXMLDocument.java:242)
	at org.hl7.fhir.r5.renderers.spreadsheets.SpreadsheetGenerator.finish(SpreadsheetGenerator.java:78)
	at org.hl7.fhir.igtools.publisher.Publisher.generateOutputsStructureDefinition(Publisher.java:9459)
	at org.hl7.fhir.igtools.publisher.Publisher.generateResourceHtml(Publisher.java:8348)
	at org.hl7.fhir.igtools.publisher.Publisher.generateHtmlOutputs(Publisher.java:8223)
	at org.hl7.fhir.igtools.publisher.Publisher.generate(Publisher.java:5991)
	at org.hl7.fhir.igtools.publisher.Publisher.createIg(Publisher.java:1018)
	at org.hl7.fhir.igtools.publisher.Publisher.execute(Publisher.java:855)
	at org.hl7.fhir.igtools.publisher.Publisher.main(Publisher.java:10193)
```

## Reproducing Example

CI runs in this repository fail with the affected versions: https://github.com/FHIR-Vaccination-Pass/fhir-implementation-guide
IG Publisher can be invoked by running `./ig/_genonce.sh` or running the gradle task `:ig:ig`.

**Comment by battisj** [comment]
This is also an issue with Windows

**Comment by dotasek** [comment]
This should be fixed in this PR: https://github.com/HL7/fhir-ig-publisher/pull/465 and available in version 1.1.125

**Comment by n-krueger** [comment]
@dotasek I can confirm that 1.1.125 fixes this issue in our project, thank you for the quick turnaround!
